ports/geckolib/values.rs

```diff
@@ -157,6 +164,7 @@ pub trait GeckoStyleCoordConvertible : Sized {
 
 impl GeckoStyleCoordConvertible for LengthOrPercentage {
     fn to_gecko_style_coord(&self, unit: &mut nsStyleUnit, union: &mut nsStyleUnion) {
```

Comment:
If the functions on `nsStyleUnion` are unsafe because they require particular `nsStyleUnit` references to be passed in, shouldn't that also mean `to_gecko_style_coord` and `from_gecko_style_coord` are unsafe too?  Or do I misunderstand the use of unsafe on the `nsStyleUnion` functions.